apps/api/src/routes/auth/test.ts

```diff
@@ -0,0 +1,18 @@
+import { env } from "@playfulprogramming/common";
+import type { FastifyPluginAsync } from "fastify";
+
+export const testAuthRoute: FastifyPluginAsync = async (fastify) => {
+	fastify.get("/api/auth/test", async (request, _reply) => {
```

Comment:
Could this route be moved to a test file & validated with unit tests instead of being added to the prod server?

You'll likely need to mock a `HOOF_AUTH_TOKEN` in [test-utils/setup.ts](https://github.com/playfulprogramming/hoof/blob/main/apps/api/test-utils/setup.ts#L34) and then write a unit test [like this](https://github.com/playfulprogramming/hoof/blob/main/apps/api/src/routes/tasks/post-images.test.ts#L6) (adding the ratelimit plugin & test route with `app.register` in each test) to get that to work.

(Also fyi - the API tests can be individually run with `pnpx nx test api`. Nx makes this a bit difficult to figure out)